app/direct4b.hs

```diff
@@ -128,20 +131,18 @@ observe = do
     ( Direct.defaultConfig
       { Direct.notificationHandler = showNotification
       , Direct.requestHandler = showRequest
+      , Direct.logger = printMsg
       }
     )
     (\_ ->
       -- `forever $ return ()` doesn't give up control flow to the receiver thread.
       forever $ threadDelay $ 10 * 1000
     )
     where
-      showNotification _c method params =
-        putStrLn $ "Notification method: " ++ show method ++ ", params: " ++ show params
+      showNotification _c _method _params = return ()
```

Comment:
Now `showNotification` and `showRequest` are same as the default values.
So delete them.

Reply:
Done.